packages/app/src/pages/dashboard/endpoints/_/body/item/index.tsx

```diff
@@ -256,3 +256,24 @@ const _Item: React.FC<{
     </>
   );
 };
+
+const SkeltonItem: React.FC<BaseProps> = ({ on }) => {
+  return (
+    <div className="flex flex-col h-full animate-pulse">
+      <div className="flex justify-between items-center gap-2.5">
+        <div className={`w-10 h-10 bg-thm-on-${on}-slight rounded`} />
+        <div className={`w-full h-6 flex-1 bg-gray-200 rounded-full`} />
```

Comment:
nits
I prefer "rounded" to "rounded-full" for text.
```suggestion
        <div className={`w-full h-6 flex-1 bg-gray-200 rounded`} />
```

Reply:
UI library often seems to use "rounded" for text so adopt suggestion.